### PR TITLE
(fix) HSC-291: Replace existing drug concepts with the HSC Drugs

### PR DESCRIPTION
**Pull request by vasharma05**
Link to the ticket: https://mekomsolutions.atlassian.net/browse/HSC-291

### CODE REVIEW

**Comment by vasharma05** [comment]
image

This drug was added in the new file and is present in the local run server.

**Review by kazlaw — COMMENTED** [review]
Did you face this @vasharma05 

[Screencast from 23-10-24 09:39:02.webm](https://github.com/user-attachments/assets/3e60b8b7-cc23-4777-995b-b15bfc877e7d)

**Comment by vasharma05** [comment]
This is because there is no "Order" Encounter type present in the HSC server.
Please look at it, since it's present for some time now

**Review by kazlaw — APPROVED** [review]
Looks Good

File: configs/openmrs/initializer_config/drugs/drugs.csv [hunk 1]
```diff
@@ -287,35 +296,36 @@ c109ae6a-9002-43ce-87c8-b1d29d5dd661,,Eucalyptus & Diphenhydramine & Phenylephri
...
-42ffb110-55a8-40e2-be2a-50aa8cd22ea3,,Furosemide 10mg/ml,9f6c3927-2a31-485c-8735-f10b9951aef9,16b302c7-715f-46cb-b328-c91afad4ccc8,10 mg/ml,
 811c8fc6-f5da-44cc-b995-9f08afc6df02,,Genfibrozil 300mg,909ea570-031b-422e-b28d-17b7f492a1b9,164cd72a-b50d-41f5-b534-7baff3e434b9,300mg,
+7a0e124d-0d98-46e3-ab09-9a91c7355e9c,,Genfibrozil 600mg Capsule,909ea570-031b-422e-b28d-17b7f492a1b9,164cd72a-b50d-41f5-b534-7baff3e434b9,600mg,
 f0b26c45-7b49-4538-85c0-e4eaa41d370d,,Gentamicin 0.10%,9019b2e5-62fd-4238-b448-909a44f845ca,0689bb6e-9371-4a0f-ad1c-63c135b1714d,0.10%,
 99822f50-b126-4311-8c1e-db511ba3aa34,,Glucosamine & chondroitin 500mg/400mg,1d47e474-4ae6-4e47-9b69-2ce6ed7399ac,fdbe907b-6abc-41dc-9599-2d84105e4063,500mg/400mg,
 8fb6321e-a67b-4169-9ef5-ec5bc526727c,,Glucosamine & chondroitin & multivit 650mg,1d47e474-4ae6-4e47-9b69-2ce6ed7399ac,fdbe907b-6abc-41dc-9599-2d84105e4063,650mg,
-b1e3cae0-b795-4f10-a9a9-48e92740f8b5,,Glycerin bb 0.9g,a90d51dd-bd38-4534-8ff6-d363d7f11fb0,ee68c384-e3d3-493b-937d-048ff58fb134,0.9g,
+b1e3cae0-b795-4f10-a9a9-48e92740f8b5,TRUE,Glycerin bb 0.9g,a90d51dd-bd38-4534-8ff6-d363d7f11fb0,ee68c384-e3d3-493b-937d-048ff58fb134,0.9g,
```
[the remark this answers is not in the document]
Comment (kazlaw) [reply configs/openmrs/initializer_config/drugs/drugs.csv:328]: That and some other drugs were retired in the PR made [here](https://github.com/CRUDEM/openmrs-config-hsc/pull/82/files#:~:text=b1e3cae0%2Db795%2D4f10%2Da9a9%2D48e92740f8b5%2CTRUE%2CGlycerin%20bb%200.9g%2Ca90d51dd%2Dbd38%2D4534%2D8ff6%2Dd363d7f11fb0%2Cee68c384%2De3d3%2D493b%2D937d%2D048ff58fb134%2C0.9g%2C) for [HSC-240](https://mekomsolutions.atlassian.net/browse/HSC-240?focusedCommentId=44769)

**Pull request merged**